legal-api/src/legal_api/core/filing.py

```diff
@@ -246,3 +250,21 @@ def get_filings_by_status(business_id: int, status: [], after_date: date = None)
             filings.append(filing)
 
         return filings
+
+    def legal_filings(self) -> Optional[List]:
+        """Return a list of the filings extracted from this filing submission.
+
+        Returns: {
+            List: or None of the Legal Filing JSON segments.
+            }
+        """
+        if not (filing := self.json):
+            return None
+
+        legal_filings = []
+        for k in filing['filing'].keys():  # pylint: disable=unsubscriptable-object
```

Comment:
is k an id, and if so, can we name it so?

Reply:
the code is moved over from the model.Filings, and k is the key from keys().